Adamant/Modules/Settings/Notifications/NotificationsViewModel.swift

```diff
@@ -24,6 +24,7 @@ final class NotificationsViewModel: ObservableObject {
     @Published var inAppSounds: Bool = false
     @Published var inAppVibrate: Bool = true
     @Published var inAppToasts: Bool = true
+    @Published var presentBuyAndSell: Bool = false
```

Comment:
In banking this type of a dialog is usually named "upsell", maybe we can rename it here to make it more aligned with this standard?
What do you think @vovameister?

Reply:
Sure, makes sense 👍🏻